src/content_cao.cpp

```diff
@@ -1786,6 +1831,14 @@ void GenericCAO::processMessage(const std::string &data)
 	}
 }
 
+void GenericCAO::knockback(const v3f &direction, f32 time_knockback)
+{
+	m_knockback_timer = time_knockback;
```

Comment:
Ok, but this is with the current implementation, it may not be necessary with my suggested implementation, or maybe it will.
This seems unusual, why can't the player move as normal and send position as normal? This is another reason i don't like this hacky implementation.